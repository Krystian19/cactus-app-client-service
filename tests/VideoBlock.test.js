import React from 'react';
import renderer from 'react-test-renderer';

const { test, expect } = window;

test('Tests are working well', () => {
  expect(true).toBe(true);
});
