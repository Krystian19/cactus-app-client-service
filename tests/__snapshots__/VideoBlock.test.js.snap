// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests are working well 1`] = `
<div
  className="video-block"
>
  <div
    className="video-block-title"
  >
    <div
      className="video-block-title-head"
    >
      <span>
        Newest episodes
      </span>
    </div>
    <a
      className="button"
      href="/newest_episodes"
      onClick={[Function]}
      tabIndex="-1"
    >
      View all
    </a>
  </div>
  <div
    className="video-block-content"
  >
    <div
      className="anime-small-thumbnail-list"
    >
      <div
        className="anime-small-thumbnail"
        onClick={[Function]}
        onKeyPress={[Function]}
        role="menuitem"
        tabIndex={0}
      >
        <div
          className="cover"
        >
          <div
            className="loader-container"
          >
            <div
              className="loader"
            />
          </div>
          <div
            className="overlay hover_hidden darken"
          >
            <svg
              className="play"
              viewBox="0 0 24 24"
            >
              <path
                d="M8,5.14V19.14L19,12.14L8,5.14Z"
              />
            </svg>
          </div>
        </div>
        <div
          className="info"
        >
          <div
            className="title"
          >
            <div
              className="title-container"
            >
              Goblin Slayer (Season 1)
            </div>
            <div
              className="detail-container"
            >
              Ep. 12
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`VideoBlock component is rendering properly 1`] = `
<div
  className="video-block"
>
  <div
    className="video-block-title"
  >
    <div
      className="video-block-title-head"
    >
      <span>
        Newest episodes
      </span>
    </div>
    <a
      className="button"
      href="/newest_episodes"
      onClick={[Function]}
      tabIndex="-1"
    >
      View all
    </a>
  </div>
  <div
    className="video-block-content"
  >
    <div
      className="anime-small-thumbnail-list"
    >
      <div
        className="anime-small-thumbnail"
        onClick={[Function]}
        onKeyPress={[Function]}
        role="menuitem"
        tabIndex={0}
      >
        <div
          className="cover"
        >
          <div
            className="loader-container"
          >
            <div
              className="loader"
            />
          </div>
          <div
            className="overlay hover_hidden darken"
          >
            <svg
              className="play"
              viewBox="0 0 24 24"
            >
              <path
                d="M8,5.14V19.14L19,12.14L8,5.14Z"
              />
            </svg>
          </div>
        </div>
        <div
          className="info"
        >
          <div
            className="title"
          >
            <div
              className="title-container"
            >
              Goblin Slayer (Season 1)
            </div>
            <div
              className="detail-container"
            >
              Ep. 12
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
